templates/portfolios/portfolios_table.html

```diff
@@ -28,12 +28,24 @@
     {% get_obj_perms request.user for portfolio as "perms" %}
     {% if "can_grant_portfolio_owner_permission" in perms %}
     <div class="col-md-2">Owner</div>
+        <div class="col-md-2">
+            <span class="glyphicon glyphicon-pencil"></span>&nbsp;&nbsp;&nbsp;
+            <a id="new-project" class="portfolio-project-link" onclick="show_manage_portfolio_modal('{{portfolio.id}}'); return false;" href="#">Manage Portfolio</a>
+        </div>
     {% elif "change_portfolio" in perms%}
     <div class="col-md-2">Portfolio Member</div>
+    <div class="col-md-2">
+        <div class="col-md-2">
+            <span class="glyphicon glyphicon-pencil"></span>&nbsp;&nbsp;&nbsp;
```

Comment:
I'm not a big fan of the use of &nbsp; to add space, but I know we're doing that throughout this project. I think that is getting swapped out though, to be replaced by CSS styling code, so if this could be replaced by spacing in CSS, that'd be great.
Same comment throughout, in a few spots.

Reply:
Not seeing an easy way to do that since we have 3 different member types Owner, portfolio member, project member

Reply:
The last comment may have been intended somewhere else. Here, I'm just mentioning adding styling code to the icon instead of the "&nbsp;"

Reply:
The last comment may have been intended under a different issue, but this issue seems to have been resolved, so I'm resolving this one.